### ISSUE TITLE
Multiple solr cores in one solr service

### ISSUE THREAD

**Issue by mrtronje** [lando/lando#2408]
Is there a possability to use multiple cores in one Solr Service? Our CMS use this way of configuration for multiple languages (one solr core per language). 

I found this comment, but it doesn't give a hint how to achieve that: 

@tanc is correct about the URLS. 
it *might* be possible to support multiple cores in a single image but that should be a separate issue

_Originally posted by @pirog in https://github.com/lando/lando/issues/1123#issuecomment-453798883_

**Comment by pirog** [comment]
@mrtronje i dont think we will have this baked into the Lando config itself but we can probably provide a guide around how you can do this manually with lando build steps

**Comment by mrtronje** [comment]
@pirog if you could provide a guide that would be great 😃

**Comment by gitressa** [comment]
Thanks Stalebot, but this issue is still relevant.
